sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -333,202 +340,11 @@ foreach (RecognizedForm receipt in receipts)
-
-                        Console.WriteLine($"  Description: '{itemDescription}', with confidence {itemDescriptionField.Confidence}");
-                    }
-                }
-
-                if (itemFields.TryGetValue("UnitPrice", out FormField itemUnitPriceField))
-                {
-                    if (itemUnitPriceField.Value.ValueType == FieldValueType.Float)
-                    {
-                        float itemUnitPrice = itemUnitPriceField.Value.AsFloat();
-
-                        Console.WriteLine($"  UnitPrice: '{itemUnitPrice}', with confidence {itemUnitPriceField.Confidence}");
-                    }
-                }
-
-                if (itemFields.TryGetValue("Quantity", out FormField itemQuantityField))
-                {
-                    if (itemQuantityField.Value.ValueType == FieldValueType.Float)
-                    {
-                        float quantityAmount = itemQuantityField.Value.AsFloat();
-
-                        Console.WriteLine($"  Quantity: '{quantityAmount}', with confidence {itemQuantityField.Confidence}");
-                    }
-                }
-
-                if (itemFields.TryGetValue("Amount", out FormField itemAmountField))
-                {
-                    if (itemAmountField.Value.ValueType == FieldValueType.Float)
-                    {
-                        float itemAmount = itemAmountField.Value.AsFloat();
-
-                        Console.WriteLine($"  Amount: '{itemAmount}', with confidence {itemAmountField.Confidence}");
-                    }
-                }
-            }
-        }
-    }
-}
-
-if (invoice.Fields.TryGetValue("SubTotal", out FormField subTotalField))
-{
-    if (subTotalField.Value.ValueType == FieldValueType.Float)
-    {
-        float subTotal = subTotalField.Value.AsFloat();
-        Console.WriteLine($"Sub Total: '{subTotal}', with confidence {subTotalField.Confidence}");
-    }
-}
-
-if (invoice.Fields.TryGetValue("TotalTax", out FormField totalTaxField))
-{
-    if (totalTaxField.Value.ValueType == FieldValueType.Float)
-    {
-        float totalTax = totalTaxField.Value.AsFloat();
-        Console.WriteLine($"Total Tax: '{totalTax}', with confidence {totalTaxField.Confidence}");
-    }
-}
-
-if (invoice.Fields.TryGetValue("InvoiceTotal", out FormField invoiceTotalField))
-{
-    if (invoiceTotalField.Value.ValueType == FieldValueType.Float)
-    {
-        float invoiceTotal = invoiceTotalField.Value.AsFloat();
-        Console.WriteLine($"Invoice Total: '{invoiceTotal}', with confidence {invoiceTotalField.Confidence}");
-    }
-}
-```
-For more information and samples see [here][recognize_invoices].
-
+For more information and samples using prebuilt models see:
+- [Receipts sample][recognize_receipts].
+- [Business Cards sample][recognize_business_cards].
+- [Invoices][recognize_invoices].
+<!-- - [ID Documents][recognize_id_documents].-->
```

Comment:
CI is mad because the links are not resolving... so comment for now and will enable in a future PR :|